kymatio/scattering3d/backend/numpy_backend.py

```diff
@@ -47,10 +47,10 @@ def compute_integrals(cls, input_array, integral_powers):
                 Tensor of size (B, P) containing the integrals of the input_array
                 to the powers p (l_p norms).
         """
-        integrals = cls._np.zeros((input_array.shape[0], len(integral_powers)),
-                dtype=cls._np.float32)
+        integrals = []
         for i_q, q in enumerate(integral_powers):
-            integrals[:, i_q] = (input_array ** q).reshape((input_array.shape[0], -1)).sum(axis=1)
+            integrals.append((input_array ** q).reshape((input_array.shape[0], -1)).sum(axis=1))
```

Comment:
is this modification to accommodate jax, because jax doesnt allow setting subarrays?
this will likely double the memory usage - worth considering whether to override this only in jax and keep numpy as it was?